index.html

```diff
@@ -1,20 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<title>3d example using three.js and multiple windows</title>
-	<script type="text/javascript" src="three.r124.min.js"></script>
-	<style type="text/css">
-		
-		*
-		{
-			margin: 0;
-			padding: 0;
-		}
+  <head>
+    <title>3d example using three.js and multiple windows</title>
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@your_twitter_handle" />
```

Comment:
don't think this is needed - no such user: https://twitter.com/your_twitter_handle
most of the twitter meta tags are not needed as Twitter will use the OpenGraph ones.

Reply:
Absolutely right.  Good point.

Reply:
`<meta name="description" content="System developed by Bjorn Staal using local storage and the getScreenDetails() API to communicate between two windows rendering webGL 3D canvas." />` could be useful to add too :-)

Reply:
Will do.

Reply:
**Twitter Card:**

<img width="475" alt="Twitter_Card" src="https://github.com/bgstaal/multipleWindow3dScene/assets/6116865/7b4a33b4-12b9-4642-bb49-363bf74a8b4c">

**Slack Preview:**

<img width="741" alt="Slack_Preview" src="https://github.com/bgstaal/multipleWindow3dScene/assets/6116865/016f1425-1e17-40b3-a668-e2415cc70219">

**Discord Preview:**

<img width="569" alt="Discord_Preview" src="https://github.com/bgstaal/multipleWindow3dScene/assets/6116865/f8f9599a-6ee3-4ee9-a132-982a63bfd802">